core/dbt/adapters/sql/connections.py

```diff
@@ -50,16 +53,25 @@ def add_query(self, sql, auto_begin=True, bindings=None,
 
         with self.exception_handler(sql):
             if abridge_sql_log:
-                logger.debug('On %s: %s....', connection.name, sql[0:512])
+                log_sql = sql[:512]
             else:
-                logger.debug('On %s: %s', connection.name, sql)
+                log_sql = sql
+
+            logger.debug(
+                'On {connection_name}: {sql}....',
```

Comment:
this is super minor, but we previously only appended ellipses if the sql was abridged. Can we preserve that behavior here? The benefit is just that the logged sql can be copy/pasted as-is from the logs